ci/check-params-env.sh

```diff
@@ -407,66 +460,74 @@ function check_image() {
-        ret_code=1
-        continue
-    }
+process_file() {
+    ret_code=0
+    while IFS= read -r LINE; do
+        echo "Checking format of: '${LINE}'"
+        [[ "${LINE}" = *[[:space:]]* ]] && {
+            echo "ERROR: Line contains white-space and it shouldn't!"
+            echo "--------------------------------------------------"
+            ret_code=1
+            continue
+        }
+        [[ "${LINE}" != *=* ]] && {
+            echo "ERROR: Line doesn't contain '=' and it should!"
+            echo "----------------------------------------------"
+            ret_code=1
+            continue
+        }
 
-    IMAGE_VARIABLE=$(echo "${LINE}" | cut --delimiter '=' --field 1)
-    IMAGE_URL=$(echo "${LINE}" | cut --delimiter '=' --field 2)
+        IMAGE_VARIABLE=$(echo "${LINE}" | cut --delimiter '=' --field 1)
+        IMAGE_URL=$(echo "${LINE}" | cut --delimiter '=' --field 2)
 
-    test -n "${IMAGE_VARIABLE}" || {
-        echo "ERROR: Couldn't parse image variable - got empty value!"
-        echo "-------------------------------------------------------"
-        ret_code=1
-        continue
-    }
+        test -n "${IMAGE_VARIABLE}" || {
+            echo "ERROR: Couldn't parse image variable - got empty value!"
+            echo "-------------------------------------------------------"
+            ret_code=1
+            continue
+        }
 
-    test -n "${IMAGE_URL}" || {
-        echo "ERROR: Couldn't parse image URL - got empty value!"
-        echo "--------------------------------------------------"
-        ret_code=1
-        continue
-    }
+        test -n "${IMAGE_URL}" || {
+            echo "ERROR: Couldn't parse image URL - got empty value!"
+            echo "--------------------------------------------------"
+            ret_code=1
+            continue
+        }
+
+        check_image "${IMAGE_VARIABLE}" "${IMAGE_URL}" || {
+            echo "ERROR: Image definition for '${IMAGE_VARIABLE}' isn't okay!"
+            echo "------------------------"
+            ret_code=1
+            continue
+        }
+    done < "${1}"
+}
 
-    check_image "${IMAGE_VARIABLE}" "${IMAGE_URL}" || {
-        echo "ERROR: Image definition for '${IMAGE_VARIABLE}' isn't okay!"
+for file_ in  "${PARAMS_ENV_PATH}" "${PARAMS_LATEST_ENV_PATH}"; do
+    echo "Checking file: '${file_}'"
+    process_file "${file_}"
+    if test "${ret_code}" -eq 0; then
+        echo "Validation of '${PARAMS_LATEST_ENV_PATH}' was successful! Congrats :)"
         echo "------------------------"
-        ret_code=1
-        continue
-    }
-done < "${PARAMS_ENV_PATH}"
-
-echo ""
-if test "${ret_code}" -eq 0; then
-    echo "Validation of '${PARAMS_ENV_PATH}' was successful! Congrats :)"
-else
-    echo "The '${PARAMS_ENV_PATH}' file isn't valid, please check above!"
-fi
+    else
+        echo "The '${PARAMS_LATEST_ENV_PATH}' file isn't valid, please check above!"
+        echo "------------------------"
+    fi
+done
 
 exit "${ret_code}"
```

Comment:
we need to be sure that `ret_code` of the second file processing doesn't override the eventual error in the first file processing - so maybe we should store this value